apps/bluetooth/message.html

```diff
@@ -17,9 +17,9 @@
     <script type="application/javascript" src="shared/js/l10n.js"></script>
 
     <!-- Specific code -->
-    <script type="application/javascript" defer src="shared/js/bluetooth_helper.js"></script>
-    <script type="application/javascript" defer src="js/pair_manager.js"></script>
-    <script type="application/javascript" defer src="js/pair_expired_dialog.js"></script>
+    <!-- <script type="application/javascript" defer src="shared/js/bluetooth_helper.js"></script> -->
+    <!-- <script type="application/javascript" defer src="js/modules/pair_manager.js"></script> -->
```

Comment:
Only the line for `bluetooth_helper.js` is required because we need it to make the build script copy the share script. For other two script they are already in the folder we don't need the last two lines.